t.py

```diff
@@ -0,0 +1,86 @@
+#coding=utf-8
+
```

Comment:
Re-read this post: https://blog.louie.lu/2017/03/21/%E5%A6%82%E4%BD%95%E5%AF%AB%E4%B8%80%E5%80%8B-git-commit-message/

Your commit title isn't that good.